libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -17,6 +17,7 @@ userdata Location method get_vector_from_origin_NEU boolean Vector3f'Null
 userdata Location method get_bearing float Location
 userdata Location method get_distance_NED Vector3f Location
 userdata Location method get_distance_NE Vector2f Location
+userdata Location method change_alt_frame boolean Location::AltFrame'enum Location::AltFrame::ABSOLUTE Location::AltFrame::ABOVE_TERRAIN
```

Comment:
What about the other AltFrames? Just wondering why not include them as well? Space & usefulness?

Reply:
Its a range, the type is `Location::AltFrame'enum` the min value we can have is `Location::AltFrame::ABSOLUTE` (0) and the max value is `Location::AltFrame::ABOVE_TERRAIN` (3) So we have to give a number >= 0 and <= 4

Reply:
Thanks for the that! Learning lua! I was wondering as the first enums I hit below it were explicitly listed.